docs/scripting/scripts.lib/is_not_today.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: scripts.lib().is_not_today()
+---
+
+## A function that checks if a specified date is not today's date.
+
+## Syntax
+
+```js
+#fs.scripts.lib().is_not_today(date);
+```
+
+### Parameters
+
+# date
+
+A JavaScript `Date` object.
+
+### Return
+
+## Example
+
+```js
+function(context, args) {
+  const l = #fs.scripts.lib();
+  const date = new Date();
+
+  return l.is_not_today(date);
```

Comment:
this will always return false. Change the example so that it returns both true and false. People will understand better then.